pkg/tests/apis/folder/folders_test.go

```diff
@@ -298,7 +298,8 @@ func doFolderTests(t *testing.T, helper *apis.K8sTestHelper) *apis.K8sTestHelper
 			"metadata": {
 			  "annotations": {
 				"grafana.app/originPath": "${originPath}",
-				"grafana.app/originName": "SQL"
+				"grafana.app/originName": "SQL",
+				"grafana.app/slug":"test"
```

Comment:
with luck... slugs will go away soon :)

If it gets complicated/weird to support them -- we can ignore them (we can always re-create it from the name)

Reply:
Nice. We rely on slugs indirectly when creating folders, because the frontend needs the URL to be set. https://github.com/grafana/grafana/blob/ff50e3689c2e83144f1ffbbf2caa94e14dcb2308/pkg/registry/apis/folders/conversions.go#L184-L188

Would you suggest replacing the `grafana.app/slug` annotation with a URL annotation?